### PR TITLE
Fixing Documentation

### LINKED ISSUE #1125: Add note that TARDIS only works on linux/mac
Add this in the documentation

### PR DESCRIPTION
**Pull request by shreyansjain012**
1. Added Note for compatible Operating systems in the documentation. #1125 
2. Corrected syntax in installation.rst
3. Updates Azure test to support macOS-10.14 and updated the documentation related to it.
## Description

- Command `python setup.py build` is wrongly highlighted as `python setup.py` build - in installation.rst
- Mac tests on Azure pipeline because macOS-10.13 was depricated and instead macOS-10.15 is introduced in Azure so updated the test to support macOS-10.15
## Screenshots (if appropriate):
1
image
image

2
Screenshot from 2020-04-11 13-51-52
update

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by shreyansjain012** [comment]
@jordi5 @jaladh-singhal @wkerzendorf - Please have a review.

File: azure-pipelines/simple_test_framework.yml [hunk 1]
```diff
@@ -28,7 +28,9 @@ jobs:
         vm_Image: 'Ubuntu-16.04'
         conda: '/usr/share/miniconda'
       mac:
+
```
Comment (jaladh-singhal) [inline azure-pipelines/simple_test_framework.yml:31]: I'm not sure why these newlines are for?

File: docs/development/continuous_integration.rst [hunk 2]
```diff
@@ -203,6 +203,7 @@ After defining the variable, one could optionally encrypt it using this lock sym
 Variables are referenced as $(variable_name), as seen in the InstallSSHKey@0 task in the hostName and sshPublicKey inputs.
 
 Azure provides a list of agent hosts that can run the pipeline on a virtual machine. In our pipelines, we
+
```
Comment (jaladh-singhal) [inline docs/development/continuous_integration.rst:206]: Same goes here?

File: docs/installation.rst [hunk 3]
```diff
@@ -18,6 +18,9 @@ simple TARDIS calculations.
 
     TARDIS is only compatbile with Python >3.6
 
+.. note::
+    TARDIS only works for Linux and Mac users.
```
Comment (jaladh-singhal) [inline docs/installation.rst:22]: IMO it can be better rephrased as "Currently, TARDIS is compatible only with LINUX and MacOS."

**Comment by jaladh-singhal** [comment]
> Updates Azure test to support macOS-10.14 and updated the documentation related to it.

@shreyansjain012 It's already showing macOS-10.14, what exactly are you changing - see the file diff?

File: docs/development/continuous_integration.rst (hunk 2 above)
Comment (shreyansjain012) [reply docs/development/continuous_integration.rst:206]: ok. I'll remove them.

**Comment by jordi5** [comment]
> > Updates Azure test to support macOS-10.14 and updated the documentation related to it.
> 
> @shreyansjain012 It's already showing macOS-10.14, what exactly are you changing - see the file diff?

I think this PR is prior to the one that added support for 10.14

File: docs/installation.rst (hunk 3 above)
Comment (shreyansjain012) [reply docs/installation.rst:22]: Thanks for the review. Let @wkerzendorf have a say about it. I'll change accordingly.

**Pull request closed without merging**